cli/components/selector.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package components
+
+import (
+	"fmt"
+	"os"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/rs/zerolog/log"
+)
+
+// Selector is the interface that items need to implement so that we can select them.
+type Selector interface {
+	HumanName() string
```

Comment:
nit: weird name. Perhaps call it `FriendlyName` or similar.
Would be nice to add documentation what this function is used for

Reply:
I didn't want to re-implement the function for the assets, since it already exists and, it is the only place we use it (for now).

If we are ok with a new function, then I'll go with `Display()`